presto-hive/src/test/java/com/facebook/presto/hive/AbstractTestHiveClient.java

```diff
@@ -3201,7 +3226,7 @@ private void doCreateView(SchemaTableName viewName, boolean replace)
         }
     }
 
-    protected void doCreateTable(SchemaTableName tableName, HiveStorageFormat storageFormat)
+    protected void doCreateTable(SchemaTableName tableName, HiveStorageFormat storageFormat, PageSinkProperties pageSinkProperties)
```

Comment:
nit: passing `PageSinkProperties pageSinkProperties` seems to be unnecessary here. How about a simple boolean flag, `boolean partitionCommitRequired`

Reply:
We need to pass `PageSinkProperties` for `pageSinkProvider.createPageSink`. Are you suggesting that we use a boolean flag and construct `PageSinkProperties` in the `doFoo` method?

Reply:
> We need to pass PageSinkProperties for pageSinkProvider.createPageSink

I missed that. Nevermind.